pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -93,6 +96,70 @@ func Convert_v1_PodSpec_To_api_PodSpec(in *v1.PodSpec, out *api.PodSpec, s conve
 	return v1.Convert_v1_PodSpec_To_api_PodSpec(in, out, s)
 }
 
+func convertToMapSelector(selector *string) (map[string]string, error) {
```

Comment:
all the string to labelselector conversions should already be available... aren't we doing that for ReplicaSet and Deployments already?